selenium/src/test/java/au/org/emii/portal/tests/landing/AcknowledgementTest.java

```diff
@@ -0,0 +1,17 @@
+package au.org.emii.portal.tests.landing;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+public class AcknowledgementTest extends BaseTest {
+
+    @Test
+    public void linkTest() {
+        webElementUtil.clickLinkContainingText("Acknowledgement");
```

Comment:
It is definitely possible to do that, but it defeats the purpose of black box testing. Functional testing is from user point of view and therefore I think it is not a good idea to bind it with the project config.  

I am happy to have a chat with you regarding the same and we can find a solution together.